attachment_swift/tests/test_mock_swift_api.py

```diff
@@ -67,10 +69,16 @@ def test_connection(self, mock_swift_client):
         session = kwargs['session']
         self.assertTrue(isinstance(session, keystoneauth1.session.Session))
         self.assertEqual(session.auth.auth_url, os.environ['SWIFT_AUTH_URL'])
-        self.assertEqual(session.auth.username, os.environ['SWIFT_ACCOUNT'])
-        self.assertEqual(session.auth.password, os.environ['SWIFT_PASSWORD'])
-        self.assertEqual(session.auth.tenant_name,
-                         os.environ['SWIFT_TENANT_NAME'])
+        self.assertEqual(session.auth.get_cache_id_elements().get(
+            'password_username'), os.environ['SWIFT_ACCOUNT'])
+        self.assertEqual(session.auth.get_cache_id_elements().get(
+            'password_password'), os.environ['SWIFT_PASSWORD'])
+        self.assertEqual(session.auth.project_name,
+                         os.environ['SWIFT_TENANT_NAME']):
```

Comment:
looks like a copy-paste ;)

Reply:
Sorry, I didn't understand your proposal at first because i am null in python, but i will correct it now. And thanks for your review and patience

Reply:
Sorry to bother, but what do you think now?